### PR TITLE
Task/fix struct reduce

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary 

- This PR is a bugfix to get OpenMP target file to compile.

### CODE REVIEW

File: scripts/lc-builds/blueos_xl_omptarget.sh [hunk 1]
```diff
@@ -37,8 +37,9 @@ cmake \
   -DBLT_CXX_STD=c++14 \
   -C ${RAJA_HOSTCONFIG} \
   -DENABLE_OPENMP=On \
-  -DENABLE_TARGET_OPENMP=On \
-  -DOpenMP_CXX_FLAGS="-qoffload;-qsmp=omp;-qalias=noansi" \
+  -DRAJA_ENABLE_TARGET_OPENMP=On \
+  -DBLT_OPENMP_COMPILE_FLAGS="-qoffload;-qsmp=omp;-qalias=noansi" \
+  -DBLT_OPENMP_LINK_FLAGS="-qoffload;-qsmp=omp;-qalias=noansi" \
   -DCMAKE_INSTALL_PREFIX=../install_${BUILD_SUFFIX} \
```
Comment (CRobeck) [inline scripts/lc-builds/blueos_xl_omptarget.sh:43]: Probably a bit pedantic but should these build script changes be in a separate PR?

Comment (rhornung67) [reply scripts/lc-builds/blueos_xl_omptarget.sh:43]: That would be best. I'm trying to reduce CI throughput issues and the script changes don't affect CI testing.

**Review by CRobeck — APPROVED** [review]
Looks good. Changes seem pretty benign.

**Review by MrBurmark — APPROVED** [review]
